src/utils/logs.py

```diff
@@ -15,7 +15,8 @@ def __init__(self, config, exp_type):
         self.curr_iters = 0
         self.total_epochs = config["total_epochs"]
         self.items = {}
-        self.metrics = MetricLoader()
+        self.metrics = MetricLoader() 
```

Comment:
Can you fix the code inside metricloader to work with cpu?

Reply:
Sure, for me passing the device argument as "CPU" worked. I have commented that in line 19. Are you suggesting changing things inside the metric loader so that the onus isn't on the user to pass this argument?